setenv.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+rm -rf /tmp/aws
+cp -R /root/.aws /tmp/.aws
+
+data="$(aws sts assume-role --role-arn arn:aws:iam::395594542180:role/product-contractors-role --profile lfproduct --role-session-name lfproduct-dev-session)"
```

Comment:
@lukaszgryglicki you happen to use your iam arn role details here

Reply:
Arent the scripts abit of work. We have collections that handle the endpoints. Weve moved away from postman but theres a recommendation to use bruno https://www.usebruno.com/ as a replacement

Reply:
This ARN is for "dev" when any of our users is assuming lfproduct-dev role - I can make that an env variable if needed.

Well I use `curl` locally, those scripts are not part of any CI or deployment, so I woudl like to have them in the repo, is this an issue? cc @mlehotskylf 

I can remove that ARN and make it a required ENV variable, but, again, it is not a secret, it is just the `dev product-contractors-role`...

Reply:
Anyway, changed to get `ARN` from the `.secret` file (`*.secret` files are gitignored) PTAL again, thanx.